controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -463,3 +469,42 @@ func (r *DSCInitializationReconciler) watchAuthResource(ctx context.Context, a c
 
 	return nil
 }
+
+func (r *DSCInitializationReconciler) configureMonitoring(ctx context.Context, dsci *dsciv1.DSCInitialization) error {
+	// Create Monitoring CR singleton
+	defaultMonitoring := client.Object(&serviceApi.Monitoring{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       serviceApi.MonitoringKind,
+			APIVersion: serviceApi.GroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: serviceApi.MonitoringInstanceName,
+			OwnerReferences: []metav1.OwnerReference{{
+				APIVersion: dsciv1.GroupVersion.String(),
+				Kind:       dsci.Kind,
+				Name:       dsci.Name,
+				UID:        dsci.UID,
+			},
+			},
+		},
+		Spec: serviceApi.MonitoringSpec{
+			MonitoringCommonSpec: serviceApi.MonitoringCommonSpec{
+				Namespace: dsci.Spec.Monitoring.Namespace,
+			},
+		},
+	},
+	)
+
+	if dsci.Spec.Monitoring.ManagementState == operatorv1.Managed {
+		err := r.Create(ctx, defaultMonitoring)
```

Comment:
ideally this should use `Apply` or `CreateOrUpdate`  since monitoring namespace is inherit from the DSCI instance

Reply:
addressed in https://github.com/opendatahub-io/opendatahub-operator/pull/1520